lib/heex_formatter/phases/format.ex

```diff
@@ -169,7 +193,14 @@ defmodule HeexFormatter.Phases.Format do
   #    iex> indent_expression(2)
   #    "  "
   defp indent_expression(indentation) do
-    String.duplicate(@tab, indentation)
+    String.duplicate(@tab, max(0, indentation))
+  end
+
+  defp indent_script(indent, text) do
+    text
+    |> String.split("\n")
+    |> Enum.map_join("\n" <> indent, &String.trim_trailing/1)
```

Comment:
```suggestion
    |> Enum.map_join("\n", &String.trim_trailing/1)
```

We can't indent JS code like that otherwise, every time you run the formatter in the same file will push the JS code further:

![2022-01-22 16 59 20](https://user-images.githubusercontent.com/27698968/150643756-6e39556a-75cb-4b7d-9463-a104155d19cb.gif)

On the other hand, if we don't add we are not formatting the JS code at all. We need to figure out another way.

Reply:
I think, best would be to not reindent between the script tags - I will have a look after dinner

Reply:
I agree. That would be code if we go this way: https://github.com/feliperenan/heex_formatter/pull/21/files#r790156679

Reply:
done